test/models/config_test.rb

```diff
@@ -19,6 +44,27 @@ class ConfigTest < ActiveSupport::TestCase
       refute @bad_config.valid?
       assert @bad_config.errors.messages[:config_key].include? "can't be blank"
     end
+
+    it 'should validate URLs' do
+      c = Config.find_or_create_by(config_key: 'fax_service')
```

Comment:
Can we do this for `practical_support_guidance_url` as well? Looks like `resource_url` and `fax_service` are both covered.

A fun experiment if you're feeling wild - test all three using a loop.